Scenes/NPC/Brute/EnemyBrute.gd

```diff
@@ -10,12 +10,15 @@ func _physics_process(_delta):
 	velocity = direction * movementSpeed
 	move_and_slide()
 
-func DamageEnemy(damage):
+func DamageEnemy(damage, incoming_dmg_pos):
```

Comment:
I think in these derived "DamageEnemy" functions we can use the "super" keyword to call the base enemy function. All the sub function need to do is define their knockback value and then call super.DamageEnemy(damage, position, knockback)